src/typings.ts

```diff
@@ -356,14 +360,33 @@ export interface DecimalsObject {
 export interface UserObject {
   address: string
   pubKey: string
-  keystore: string
+  backup: string
+}
+
+export interface DeployedIdentity {
+  identity: string
```

Comment:
is this the address? then we should call it address. It does not make much sense that a structure Identity has a identity.

Reply:
I just had it match the fields of the relay server API so I can do things like:
`const identity = await this.provider.postToEndpoint<DeployedIdentity>`
But I will change it to identityAddress.

Reply:
Actually, the only alternative solution I found is to use `postToEndpoint<any>()` which I believe is worse. So i changed the type name to `DeployIdentityResponse` and kept `identity` as a field.